config/default.yml

```diff
@@ -2980,6 +2980,16 @@ Style/HashEachMethods:
   VersionAdded: '0.80'
   Safe: false
 
+Style/HashLikeCase:
+  Description: >-
+                  Checks for places where `case-when` represents a simple 1:1
+                  mapping and can be replaced with a hash lookup.
+  Enabled: 'pending'
```

Comment:
Will this cop be enabled by default at RuboCop 1.0? I'm not sure about creating a constant of hash instance is a better style than `case` syntax if there is no performance advantage. I think it may be considered disabled by default.

Reply:
I think it most cases using the hash results in code that's simpler, as you eliminate branching completely. Obviously a simple `case` is not a big problem, but I feel that in most cases using a hash results in cleaner code.